ts-types/map/ensure.d.ts

```diff
@@ -0,0 +1,5 @@
+import { EnsureOptions } from '../ensure';
+
+declare function ensureMap<K, V>(value: any, options?: EnsureOptions): Map<K, V>;
+declare function ensureMap(value: any, options?: EnsureOptions): Map<any, any>;
```

Comment:
@medikoo and this one as well?

Reply:
Yes, with this one it's only input that can be returned.